### PR TITLE
Support setting spdk_tgt log level and flags

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue longhorn/longhorn#7939

#### Additional documentation or context

Deps on https://github.com/longhorn/go-spdk-helper/pull/76
Deps on https://github.com/longhorn/go-spdk-helper/pull/77

### CODE REVIEW

File: proto/spdkrpc/spdk.proto [hunk 1]
```diff
@@ -50,6 +50,11 @@ service SPDKService {
     rpc DiskDelete(DiskDeleteRequest) returns (google.protobuf.Empty);
     rpc DiskGet(DiskGetRequest) returns (Disk);
 
+    rpc LogSetLevel(LogSetLevelRequest) returns (google.protobuf.Empty);
+	rpc LogSetFlags(LogSetFlagsRequest) returns (google.protobuf.Empty);
+	rpc LogGetLevel(LogGetLevelRequest) returns (LogGetLevelResponse);
+	rpc LogGetFlags(LogGetFlagsRequest) returns (LogGetFlagsResponse);
```
Comment (innobead) [inline proto/spdkrpc/spdk.proto:56]: Fix the incorrect indention formatting.

File: pkg/spdk/log.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	spdkclient "github.com/longhorn/go-spdk-helper/pkg/spdk/client"
+)
+
+const (
+	ServiceSpdkTgt = "spdk_tgt"
+)
+
+func svcLogSetLevel(spdkClient *spdkclient.Client, service, level string) error {
+	log := logrus.WithFields(logrus.Fields{
+		"service": service,
```
Comment (innobead) [inline pkg/spdk/log.go:18]: Is `service` is just for info purpose? will we have any other chances of having another service other than spdk_tgt?

File: proto/spdkrpc/spdk.proto [hunk 3]
```diff
@@ -351,3 +356,29 @@ message DiskDeleteRequest {
     string disk_name = 1;
     string disk_uuid = 2;
 }
+
+message LogSetLevelRequest {
+	string service = 1;
```
Comment (innobead) [inline proto/spdkrpc/spdk.proto:361]: Is `service` required?

**Review by innobead — CHANGES_REQUESTED** [review]
In general, LGTM. Just a few comments.

File: pkg/spdk/log.go (hunk 2 above)
Comment (innobead) [reply pkg/spdk/log.go:18]: That's OK. If you think it's helpful, let's keep it.

**Comment by derekbit** [comment]
> It seems the PR was not updated yet?

Sorry, I mistakenly requested review.

File: proto/spdkrpc/spdk.proto (hunk 1 above)
Comment (derekbit) [reply proto/spdkrpc/spdk.proto]: Fixed. Thank you.

**Comment by derekbit** [comment]
Rebased and ready to review. cc @innobead

**Review by innobead — APPROVED** [review]

**Pull request merged**